Documentation/ceph-disaster-recovery.md

```diff
@@ -229,6 +229,166 @@ kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `mon.count`.
```

Comment:
There are enough recovery sections in this document, could you add links at the top of the document to the sections, similar to the links in the [ceph-common-issues.md](https://rook.github.io/docs/rook/v1.5/ceph-common-issues.html#table-of-contents-)?